isis/src/clipper/apps/jitterfit/jitterfit.xml

```diff
@@ -2,37 +2,89 @@
 
 <application name="jitterfit" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://isis.astrogeology.usgs.gov/Schemas/Application/application.xsd">
   <brief>
-    Registers lines from a normal image with lines from the check image.
+    Characterize jitter for rolling shutter images.
   </brief>
   <description>
     <p>
-      This program co-regsisters lines in the main cube (FROM) with the corrisponding line in
-      the checkline cube (FROM2).
-      The sensor line used to collect the checkline is used to center the registration around
-      the same area in the main cube.
+      This program calculates a two sets of polynomial coefficients that characterize the jitter in an
+      image acquired by a rolling shutter instrument. One polynomial for the line direction and on
+      for the sample direction. The ISIS camera model for the instrument can
+      then use the coefficients to adjust for the time dependent jitter.
```

Comment:
General comment - is this application general enough to be used for any rolling shutter instrument, or is this specific to EIS?

Reply:
Added a new paragraph